crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -83,6 +102,7 @@ pub struct AstScanner<'me, 'ast> {
   visit_path: Vec<AstKind<'ast>>,
   scope_stack: Vec<Option<ScopeId>>,
   options: Option<&'me SharedOptions>,
+  potential_update_import_reference: FxHashMap<ReferenceId, PotentialImportUpdateReference>,
```

Comment:
Oxc semantic will record the import info, so there is method like `self.result.symbol_ref_db.get_flags(symbol_id).is_import()`.

Is replacing `is_const_var` with `is_import` what this PR want

https://github.com/rolldown/rolldown/blob/bc16ada28c7122bf0246e8e3f46bb180179d6729/crates/rolldown/src/ast_scanner/mod.rs#L626-L639

Reply:
Nice catch.